apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -234,11 +256,20 @@ export default function sectionAssessments(state = initialState, action) {
 
 // Returns an array of objects, each indicating an assessment name and it's id
 // for the assessments and surveys in the current script.
-export const getCurrentScriptAssessmentList = state =>
-  computeScriptAssessmentList(
+export const getCurrentScriptAssessmentList = state => {
+  let tempAssessmentList = computeScriptAssessmentList(
     state.sectionAssessments,
     state.scriptSelection.scriptId
   );
+  /* Only add the feedback option to the dropdown for CSD and CSP */
+  if (doesCurrentCourseUseFeedback(state)) {
+    tempAssessmentList = tempAssessmentList.concat({
+      id: 0,
```

Comment:
let's move this id into a constant so that we can reference the constant in the SectionAssessments component. so in this file, we'd have something like: `export const ASSESSMENT_FEEDBACK_OPTION_ID = 0;`

then, in SectionAssessments, we can import that constant and reference it instead:
```
const isCurrentAssessmentFeedbackOption = this.props.assessmentId === ASSESSMENT_FEEDBACK_OPTION_ID;
```